statsd/client.py

```diff
@@ -82,7 +82,7 @@ def timer(self, stat, rate=1):
 
     def timing(self, stat, delta, rate=1):
         """Send new timing information. `delta` is in milliseconds."""
-        self._send_stat(stat, '%d|ms' % delta, rate)
+        self._send_stat(stat, '%f|ms' % delta, rate)
```

Comment:
Would it be worth being more specific about the format here, to make the output more predictable, especially in the tests?